Find.java

```diff
@@ -5,6 +5,10 @@
 public class Find {
   public static void main(String[] args) {
 
+    for(int i = 0; i < 10000000;i++){
```

Comment:
This is an unnecessary loop which clogs up the program, it can be removed

Reply:
So soryy papa harry I will do better next time